.scalafmt.conf

```diff
@@ -45,6 +49,24 @@ project {
 }
 
 rewrite {
+  rules = [
+    Imports
+    SortModifiers
+  ]
+  imports {
+    selectors = fold
+    removeRedundantSelectors = true
+    // with sorting and grouping, be extra careful about relative imports
+    sort = ascii
+    groups = [
+      ["sbt\\..*"]
+      ["java.?\\..*"]
+      ["scala\\..*", "dotty\\..*"]
+      ["org\\..*", "com\\..*"]
+      ["scala\\.scalanative\\..*", "scalanative\\..*"]
+      ["[a-z]+\\..*"] // catch-all for other top-level-like packages
+    ]
```

Comment:
@ekrich please check that this is the ordering of groups you prefer.

Reply:
Sure, this sounds good - not sure if others feel or how the code base is now with `scalanative` after Scala?

I guess here `sbt` is sort of a special rule or should it be somewhere after `scala`?